nix/process-compose/cli.nix

```diff
@@ -70,6 +58,27 @@ in
         default = true;
         description = "Enable or disable the TUI for the application.";
       };
+      cli = cliOptions;
+      test-cli = cliOptions;
     };
+  config = {
+    cli = lib.mkMerge [{
+      up = {
+        config = [ "${config.outputs.settingsFile}" ];
+        tui = config.tui;
+      };
+      global = {
+        port = config.httpServer.port;
+        use-uds = config.httpServer.uds != false;
+        unix-socket = if builtins.isString config.httpServer.uds then config.httpServer.uds else "";
+        no-server = if config.httpServer.enable == true then false else true;
+      };
+    }];
+    test-cli = lib.mkMerge [
```

Comment:
I can do that, but then i'll remove the "config" option as well, because it would get overridden for the test-process anyway. I think it's hard to justify that option anyway, because setting it basically eliminates the whole point of this project.